GOVERNANCE.md

```diff
@@ -18,15 +18,61 @@ This project may include the following roles. Additional roles may be adopted an
 
 ## 3. How We Work.
 
-**3.1. Openness**. Participation is open to anyone who is directly and materially affected by the activity in question. There shall be no undue financial barriers to participation.
+### 3.1. Principles
 
-**3.2. Balance**. The development process should balance the interests of Contributors and other stakeholders. Contributors from diverse interest categories shall be sought with the objective of achieving balance.
+**3.1.1. Openness**. Participation is open to anyone who is directly and materially affected by the activity in question. There shall be no undue financial barriers to participation. In that sense, discussions should prioritize public channels (Github discussions/issues/PRs or Discord) whenever possible.
 
-**3.3. Coordination and Harmonization**. Good faith efforts shall be made to resolve potential conflicts or incompatibility between releases in this Project.
+**3.1.2. Balance**. The development process should balance the interests of Contributors and other stakeholders. Contributors from diverse interest categories shall be sought with the objective of achieving balance.
```

Comment:
Do we have a definition of "Contributor"? I think it would be good to start this document with a definition, preferably an expansive one (for instance recognizing engaging in public discussion around the project as a form of contribution).